locales/en/app.ftl

```diff
@@ -200,4 +238,6 @@ copyright-info =
   Portions of this content are &#x24B8; 1998-2018 by individual mozilla.org contributors. <br />
   Content available under a  <a href="https://www.mozilla.org/foundation/licensing/website-content/" target="_blank" rel="noopener">Creative Commons license</a>.
 
+# This string is followed by a link to Have I Been Pwned, the code for which is created elsewhere.
+# The final string reads: Breach data provided by Have I Been Pwned. 
```

Comment:
Ref: 
https://github.com/mozilla/blurts-server/pull/543#discussion_r226927638

I think @flodolo was saying in the other PR that the concat isn’t great for l10n and we should do a placeholder link so localizers can rearrange where the link best fits on a per-language basis.

Reply:
Let's do the fix in https://github.com/mozilla/blurts-server/pull/543#discussion_r227007616 so we can merge this PR.